src/main/java/com/openlattice/search/SearchService.java

```diff
@@ -315,14 +310,11 @@ public void updatePropertyTypesInEntitySet( PropertyTypesInEntitySetUpdatedEvent
     }
 
     /**
-     * If 1 or more property types are added to an entity set, all the corresponding mapping need to be updated
+     * If 1 or more property types are added to an entity type, the corresponding mapping needs to be updated
      */
     @Subscribe
-    public void addPropertyTypesToEntitySet( PropertyTypesAddedToEntitySetEvent event ) {
-        elasticsearchApi.addPropertyTypesToEntitySet(
-                event.getEntitySet().getId(),
-                event.getNewPropertyTypes(),
-                event.getLinkedEntitySetIds() );
+    public void addPropertyTypesToEntityType( PropertyTypesAddedToEntityTypeEvent event ) {
+        elasticsearchApi.addPropertyTypesToEntityType( event.getEntityType(), event.getNewPropertyTypes() );
```

Comment:
I think we need linkedEntitySetIds here too, because of linked document mappings:
<entity_set_id>.<property_type_id>